web/ui/website/assets/images/sdk.svg

```diff
@@ -0,0 +1,9 @@
+</defs>
+</svg>
```

Comment:
Please replace this with a png image. Base 64 string in SVG images takes longer to load most times, and considering they'll likely open this on a phone or tab.

Reply:
okay noted